valhalla/baldr/directededge.h

```diff
@@ -3,7 +3,7 @@
 
 #include <valhalla/baldr/graphconstants.h>
 #include <valhalla/baldr/graphid.h>
-#include <valhalla/baldr/json.h>
+#include <valhalla/baldr/rapidjson_utils.h>
```

Comment:
Just a general nit that can be ignored - `valhalla/baldr/rapidjson_fwd.h` can be included instead if only forward declaration of the `rapidjson::writer_wrapper_t` is needed.

Such change would require to add `#include "baldr/rapidjson_utils.h"` in `.cc` file, but that's the idea - have forward declarations in header as they are used in many places and have implementation include only where they needed.

Ofc. we can address this later